.github/workflows/web.yml

```
@@ -601,10 +602,21 @@
           key: ${{ github.sha }}-${{ github.run_id }}-${{ steps.matrix_include.outputs.package_extension }}-${{ steps.matrix_include.outputs.distrib }}
           fail-on-cache-miss: true
 
+      - run: |
```

actionlint

shellcheck reported issue in this script: SC2035:info:8:4: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)